src/services/filterManager.js

```diff
@@ -262,14 +262,13 @@ export const rawFilterToSearchQuery = rawFilter => {
 export const toSort = filter => {
   switch (filter.active) {
     case ACTIVE_QUICK:
-      return ['_id']
-    case ACTIVE_BASIC:
-      return filter.sorting ? formatSort(filter.sorting) : ['_id']
+      return ['_uid']
     case ACTIVE_RAW:
       return filter.raw ? rawFilterToSort(filter.raw) : ['_id']
```

Comment:
is it normal that all default fields have been defaulted to `_uid`, but not this one?